testsuite/test_meeting_processing_file_list.py

```diff
@@ -0,0 +1,146 @@
+from flask import url_for
+from pyquery import PyQuery
+from rq import Queue
+from mrt.models import redis_store, Job, db
+from mrt.meetings.printouts import _process_short_list
+from .factories import RoleUserMeetingFactory
+from .factories import RoleFactory, JobsFactory
+
+
+def test_meeting_manager(app):
+    first_role = RoleFactory(permissions=('manage_meeting',))
+    second_role = RoleFactory(permissions=('view_participant',))
+    first_role_user = RoleUserMeetingFactory(role=first_role)
+    second_role_user = RoleUserMeetingFactory(role=second_role,
+                                              meeting=first_role_user.meeting)
+
+    q = Queue(Job.PRINTOUTS_QUEUE, connection=redis_store._redis_client,
+              default_timeout=1200)
+    first_job_redis = q.enqueue(_process_short_list,
+                                first_role_user.meeting.id,
+                                'List of participants', 'verified')
+    second_job_redis = q.enqueue(_process_short_list,
+                                 first_role_user.meeting.id,
+                                 'List of participants', 'verified')
```

Comment:
You don't need the `Queue` object and the redis jobs. You can use a bogus id for the JobsFactory.